src/contracts/ContractsAddreses.ts

```diff
@@ -0,0 +1,16 @@
+export const CONTRACTS_ADDRESSES: {
+  [key: string]: { LedaAddress: string; MarketplaceAddress: string };
+} = {
+  local: {
```

Comment:
Maybe we should not persist this local data, that is main point of this refactor

Reply:
The previous PR fix this problem